### PR TITLE
#9  Expand functionallity of UnitTestLibrary (3)

### PR DESCRIPTION
**Pull request by alexejFroebel**
Issue: #9 
This is a mayor release
Release Notes:
- add additional masterdata
- its now described how to install the library from github
- eval is now global, so the user does not have to eval in his code
CR Checklist:
- [x]  Does the code work?
- [x]  Is all the code easily understood?
- [x] Are the function signatures named properly (function name, parameter names?)
- [x] Is the spelling correct?

- [x]  Is there redundant or duplicate code?
- [x]  Is there any commented out code?
- [x]  Can any logging or debugging code be removed?
- [x]  Is for(i) or while used? If so, remove
- [x]  Can any of the code be replaced with library functions?

-  Do you check input parameters (such as route parameters) for:
- [x]   Type
- [x]   Length
- [x]   Format
- [x]   Range?

- [x]  Can comments be removed?
- [x] Is "external functionality" (API, Readme.md etc) documented/updated?
- [ ]  Is any unusual behaviour or edge case handling described?

- [ ]  Do tests exist?
- [ ] Do the snapshots look correct?
- [ ] Are tests comprehensive?
- [ ]  Do unit tests actually test that the code is performing the intended functionality?

### CODE REVIEW

**Comment by pflenker** [comment]
Is the examples-library also updated?

**Comment by alexejFroebel** [comment]
> Is the examples-library also updated?

No, but I made a ticket on the board, because it needs the new release number and the new way of installing it. So there would be no chance to test it, if we would do it in this task.

**Comment by pflenker** [comment]
We need to do this more or less at the same time, so we should update the unit test examples now and test if everything works, and change the package.json to the final new version as soon as everything is ready.

File: index.js [hunk 1]
```diff
@@ -5,28 +5,37 @@ const fs = require("fs")
...
+    validTo: jest.fn(),
+    validFrom: jest.fn(),
+    extProductId: jest.fn(),
+    mfgProductId: jest.fn(),
+    manufacturerName: jest.fn()
   }
   try{
-    eval(unparsedExpressions)
+    (1, eval)(unparsedExpressions);
   }catch(e){
```
[the remark this answers is not in the document]
Comment (alexejFroebel) [reply index.js]: Where? You mean between the )( ? If so, even eslint dont want a whitespace to be there

[the remark this answers is not in the document]
Comment (pflenker) [reply index.js]: No, directly in the line I commented on, between the bracket and the "catch"

[the remark this answers is not in the document]
Comment (alexejFroebel) [reply index.js]: Oh i see, but it is fixed :D

File: index.js [hunk 2]
```diff
@@ -1,32 +1,42 @@
...
+    unitOfMeasureId: jest.fn(),
+    salesUnitOfMeasureId: jest.fn(),
+    validTo: jest.fn(),
+    validFrom: jest.fn(),
+    extProductId: jest.fn(),
+    mfgProductId: jest.fn(),
+    manufacturerName: jest.fn(),
+  };
+  try {
+    // This resolves to the call: eval(unparsedExpressions). But with this way, eval is called indirect, which changes the behavior to evaluate to the global context.
```
Comment (pflenker) [inline index.js]:
```suggestion
    // This resolves to the call: eval(unparsedExpressions). But with this way, eval is called indirectly, which changes the behavior to evaluate to the global context.
```
